libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/EventQueues.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive
+{
+    public class EventQueues
+    {
+        private const string Events = "this.events";
+
+        /// <summary>
+        /// Gets mappings where a property is ready to be set to a specific entity.
+        /// </summary>
+        /// <value>List of entities to use when changeing properties.</value>
+        public List<EntityToProperty> SetProperty { get; } = new List<EntityToProperty>();
```

Comment:
This names aren't great, 1. they don't imply collection because they aren't plural 2. they conflict with function/class names. 
What about PropertyAssignments?